angular.json

```diff
@@ -29,6 +29,8 @@
             "styles": [
               "node_modules/@cds/city/css/bundles/default.min.css",
               "node_modules/@cds/core/styles/module.shims.min.css",
+              "node_modules/@cds/core/list/list.min.css",
```

Comment:
Not sure why we need to add the `<component>.min.css` here ? More of a question?

Reply:
The list and tables are not part of the Core global CSS, so they have to be independently added.